tools/aws/s3_bucket_size.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Queries Cloud Watch to get the size of each S3 bucket in the account.
+#
+set -euo pipefail
+
+YESTERDAY=$(date +%Y-%m-%d --date=yesterday)
```

Comment:
Had some trouble with this (for whatever reason) on my mac.  Does this work on your machine as well?

```YESTERDAY=$(date -v -1d '+%Y-%m-%d')```

Reply:
No doesn't work for me. I'm using GNU coreutils `date`.

Reply:
what do you think about something like this:

```if [ $( uname -s ) == "Darwin" ]
then
    YESTERDAY=$( date -v -1d '+%Y-%m-%d' )
else
    YESTERDAY=$( date +%Y-%m-%d --date=yesterday )
fi```

Reply:
updated